js/widgets/fixedToolbar.workarounds.js

```diff
@@ -73,7 +73,7 @@ define( [ "jquery", "../widget", "../core", "../navigation", "./page", "../zoom"
 		//setting the li's to -webkit-transform:translate3d(0,0,0); solves this problem to avoide potential issues in other
 		//platforms we scope this with the class ui-android-2x-fix
 		_bindListThumbWorkaround: function() {
-			this.element.closest( ".ui-page" ).addClass( "ui-android-2x-fixed" );
+			this._addClass( this.element.closest( ".ui-page" ), null, "ui-android-2x-fixed" );
```

Comment:
may need to update this with better page logic looks like it was never updated for external toolbars